internal/pkg/config/config.go

```diff
@@ -25,6 +25,12 @@ const (
 	// OperatorName is the name when referring to the operator.
 	OperatorName = "security-profiles-operator"
 
+	// OperatorRoot is the root directory of the operator.
+	OperatorRoot = "/var/lib/security-profiles-operator"
+
+	// UserRootless is the user which runs the operator.
+	UserRootless = 65535
```

Comment:
why not make this int64 already? that way you don't need to cast it later on

Reply:
`os.Chown` accepts only `int`: https://golang.org/pkg/os/#Chown

This is used within nonrootenabler.go, so I would have a need for casting there.

Reply:
gotcha. That's fine then.